src/FormBuilder/resources/js/form-builder/src/blocks/fields/login/icon.tsx

```diff
@@ -1,8 +1,18 @@
-import {Icon} from "@wordpress/icons";
+import {Icon} from '@wordpress/icons';
 
-export default () => <Icon icon={
-    <svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
-        <path d="M5.9647 0.136393C6.47635 -0.00070332 7.06701 -0.000383487 7.86239 4.72166e-05L14.2413 9.59732e-05C15.0462 8.32178e-05 15.7106 7.27288e-05 16.2518 0.0442901C16.8139 0.0902153 17.3306 0.18878 17.8159 0.43607C18.5686 0.819563 19.1805 1.43149 19.564 2.18414C19.8113 2.66947 19.9099 3.18617 19.9558 3.74827C20 4.28946 20 4.95381 20 5.75877V14.2414C20 15.0464 20 15.7107 19.9558 16.2519C19.9099 16.814 19.8113 17.3307 19.564 17.8161C19.1805 18.5687 18.5686 19.1806 17.8159 19.5641C17.3306 19.8114 16.8139 19.91 16.2518 19.9559C15.7106 20.0001 15.0463 20.0001 14.2413 20.0001L7.8624 20.0001C7.06701 20.0006 6.47635 20.0009 5.9647 19.8638C4.58433 19.4939 3.50614 18.4157 3.13627 17.0354C2.99917 16.5237 2.99949 15.9331 2.99993 15.1377V14.6001C2.99993 14.0401 2.99993 13.76 3.10892 13.5461C3.20479 13.358 3.35777 13.205 3.54593 13.1091C3.75985 13.0001 4.03987 13.0001 4.59993 13.0001H6.0404C6.41102 13.0001 6.59632 13.0001 6.70126 13.0521C6.81675 13.1094 6.85725 13.1484 6.91871 13.2617C6.97455 13.3647 6.98372 13.6135 7.00205 14.1112C7.02894 14.8415 7.32115 15.5639 7.87868 16.1214C9.05025 17.293 10.9497 17.293 12.1213 16.1214L16.1213 12.1214C16.409 11.8338 16.626 11.5023 16.7724 11.1485C16.9174 10.7988 16.9982 10.4157 17 10.014L17 10.0001L17 9.99035C16.9975 9.22584 16.7046 8.4621 16.1213 7.8788L12.1213 3.8788C10.9497 2.70723 9.05025 2.70723 7.87868 3.8788C7.32115 4.43633 7.02894 5.1587 7.00205 5.88902C6.98372 6.38672 6.97455 6.63557 6.91871 6.73852C6.85725 6.85182 6.81675 6.89086 6.70126 6.94811C6.59632 7.00012 6.41102 7.00012 6.0404 7.00012H4.59993C4.03987 7.00012 3.75985 7.00012 3.54593 6.89113C3.35777 6.79526 3.20479 6.64227 3.10892 6.45411C2.99993 6.2402 2.99993 5.96018 2.99993 5.40012V4.86252C2.99949 4.06713 2.99917 3.47647 3.13627 2.96482C3.50614 1.58445 4.58433 0.506262 5.9647 0.136393Z" fill="#0E0E0E"/>
-        <path d="M9.29289 5.29302C9.68342 4.90249 10.3166 4.90249 10.7071 5.29302L14.7071 9.29302C15.0976 9.68354 15.0976 10.3167 14.7071 10.7072L10.7071 14.7072C10.3166 15.0978 9.68342 15.0978 9.29289 14.7072C8.90237 14.3167 8.90237 13.6835 9.29289 13.293L11.5858 11.0001H1C0.447715 11.0001 0 10.5524 0 10.0001C0 9.44784 0.447715 9.00012 1 9.00012H11.5858L9.29289 6.70723C8.90237 6.3167 8.90237 5.68354 9.29289 5.29302Z" fill="#0E0E0E"/>
-    </svg>
-} />
+export default () => (
```

Comment:
Was the login icon updated in this PR?

Reply:
Yes! @jdghinson provided a new icon for the billing address block and ask to also update this one together.